src/main/java/eu/elixir/ega/ebi/it/res/ResBase.java

```diff
@@ -0,0 +1,34 @@
+package eu.elixir.ega.ebi.it.res;
+
+import java.util.InputMismatchException;
+
+import com.jayway.restassured.RestAssured;
+
+import okhttp3.Request;
+
+public class ResBase {
```

Comment:
Why is the ResBase different from the other in the sense that it does not create a REQUEST that you use in the tests?

Reply:
Maybe also add some checks to ensure the properties are not null or empty?

Reply:
Here I have used different Request class from okhttp3. It needs URL while building the object so moved the object building process to subclass.
And the extra null checks are added.

Reply:
Yeah sure, my question was more why you use this new library instead of the previous one

Reply:
It was not working correctly before with the RestAssured response InputStream. So I used okhttp3.
But today I changed it back to RestAssured and checked it again and it started working. So now I have removed the okhttp3 and requesting it using RestAssured.